### PR TITLE
Fixed errors, Adjusted mobile versions

### PR DESCRIPTION
**Pull request by TheRitual**

### CODE REVIEW

File: src/common/Vote/index.js [hunk 1]
```diff
@@ -1,25 +1,14 @@
...
-export const MovieListVote = ({ score, count }) => {
-    return (
-        <ListVoteWrapper>
-            <StarIcon src={star} />
-            <Score>{score}</Score>
-            <Count> {count} votes</Count>
-        </ListVoteWrapper>
-    );
-};
-
```
Comment (Ewelina-EN) [inline src/common/Vote/index.js:24]: Now I know another way to solve it :)

File: src/common/Vote/styled.js [hunk 2]
```diff
@@ -2,41 +2,80 @@ import styled from "styled-components";
...
-`;
+export const Count = styled.span`
+    grid-area: count;
+    font-size: ${({ type }) => type === "details" ? "14px" : "16px"};
+    white-space: nowrap;
+    color: ${({ theme, type }) => {
+            if (type === "header") { return theme.colors.voteHeaderScore }
+            if (type === "details") { return theme.colors.voteListScore }
+            return theme.colors.subtitleColor
+        }};
```
Comment (Ewelina-EN) [inline src/common/Vote/styled.js:74]: I can see a lot of changes, but it's nice that the shadows display correctly :)

File: src/features/MoviesBrowser/MovieDetailPage/MovieHeader/styled.js [hunk 3]
```diff
@@ -1,130 +1,53 @@
...
-    @media ( max-width: ${({ theme }) => theme.breakpoints.mobile}px) {
-    };
-`;
-
-export const Text = styled.article`
-    @media ( max-width: ${({ theme }) => theme.breakpoints.mobile}px) {
+    
+    @media (max-width: ${({ theme }) => theme.breakpoints.tabletLandscape}) {
+        height: auto;
+        width: 100%;
```
Comment (Ewelina-EN) [inline src/features/MoviesBrowser/MovieDetailPage/MovieHeader/styled.js:34]: I can see a lot of changes, but it's nice that the shadows display correctly :)

**Comment by Ewelina-EN** [comment]
Lots of changes and a lot of work :) Thank you for tidying up and repairing. Now I can see how some things can be done differently :)

File: src/common/Vote/styled.js [hunk 4]
```diff
@@ -2,41 +2,80 @@ import styled from "styled-components";
...
-    color: ${({ theme }) => theme.colors.linkHeader};
-`;
-
-export const Count = styled.p`
-    padding-left: 4px;
-    font-size: 14px;
-    color: ${({ theme }) => theme.colors.linkHeader};
-    margin: 0 0 0 4px;
+export const Range = styled.span`
+    display: ${({ type }) => type === "header" || type === "details" ? "inline-block" : "none"};
```
Comment (emilia82) [inline src/common/Vote/styled.js:55]: This solution is also good. And its much easier implement different type of values. Great job.

File: src/features/MoviesBrowser/MovieDetailPage/MovieHeader/styled.js [hunk 5]
```diff
@@ -1,130 +1,53 @@
...
+    max-width: ${({ theme }) => theme.breakpoints.container};
     margin: 0 auto;
     background-repeat: no-repeat;
     background-position: center;
-    background-size: contain;
-    
-    @media ( max-width: ${({ theme }) => theme.breakpoints.mobile}px) {
-    };
+    background-size: cover;
+    background-image: url('${({ image }) => image}');
```
Comment (emilia82) [inline src/features/MoviesBrowser/MovieDetailPage/MovieHeader/styled.js:15]: So now this image is well positioned in the design, great job

File: src/common/Vote/index.js (hunk 1 above)
Comment (TheRitual) [reply src/common/Vote/index.js:24]: Being honest I will make it back to separated versions for all 3 places in the project as it's easier to read code then

File: src/features/MoviesBrowser/MovieDetailPage/MovieHeader/styled.js (hunk 3 above)
Comment (TheRitual) [reply src/features/MoviesBrowser/MovieDetailPage/MovieHeader/styled.js:34]: The main problem was that the whole part should be out of `<Main>` component to reach all the width

**Pull request merged**